src/clusterfuzz/_internal/bot/fuzzers/afl/launcher.py

```diff
@@ -655,6 +706,10 @@ def generate_afl_args(self,
       -o parameter to the afl tool. Defaults to
       self.afl_output.output_directory.
 
+      target_path: Path to afl fuzz target. Defaults to self.target_path
```

Comment:
I think this comment is very confusing. `target_path` is the path to the target binary passed to `afl-fuzz` right?
as is, the comment sounds like it is talking about afl-fuzz